completion/available/vault.completion.bash

```diff
@@ -1,4 +1,5 @@
-#!/usr/bin/env bash
+#!/usr/bin/env echo run from bash: .
+# shellcheck shell=bash disable=SC2148,SC2096
```

Comment:
the `disable` part can just reside in our `.shellcheckrc`, will make the codebase prettier

Reply:
Can `.shellcheckrc` apply the disable to **just** `.bash` files?  I wouldn't want either of these disable codes to be applied to all scripts globally.

Reply:
Oh, this was a hindsight from me, they do not support it